app/Http/Controllers/Front/ContactController.php

```diff
@@ -37,7 +37,9 @@ public function create()
      */
     public function store(ContactRequest $request)
     {
-        Contact::create ($request->all ());
+        $contact = (new Contact())->fill($request->all());
```

Comment:
Create a contact, and then store it via the ContactRepository, instead of creating it and saving it all in one statement. This code is clearer, and it also gives us more control of the repo, as you'll see below.